spec/mongo/collection_spec.rb

```diff
@@ -2,8 +2,15 @@
 
 describe Mongo::Collection do
 
-  after do
-    authorized_collection.delete_many
+  before do
+    begin
+      authorized_collection.delete_many
+    rescue Mongo::Error::OperationFailure
```

Comment:
Just to make sure I understand, is the reason for catching errors here (and below) that we can't delete things from the collection while another thread is using it?

Reply:
Some collections/indexes cannot be removed when they don't exist, I have not bothered to investigate why yet.

Reply:
I changed this to drop collection and a number of tests failed.